test/GoogleMaps.js

```diff
@@ -108,6 +108,13 @@
 				expect(GoogleMapsLoader.createUrl()).to.be.equal(baseUrl + '?callback=' + cb + '&sensor=false&channel=abcdefghijkl');
 			});
 
+			it('should create url with language', function() {
+				GoogleMapsLoader.CHANNEL = undefined;
```

Comment:
No you didn't, sorry.. It's actualy a bug..

CHANNEL variable is not cleared when `release` method is [called](https://github.com/Carrooi/Node-GoogleMaps/blob/develop/lib/Google.js#L133-L137).

Later today I'll fix that, may I ask you than to rebase and update this PR please? I'll let you know when it's done.